lib/datapacktypes/omniscript.js

```diff
@@ -10,24 +10,27 @@ var OMNI_BULK_RECORDS_LIMIT_MIN = 1000;
 OmniScript.prototype.createTitleObjects = function(input) {
     var dataPackData = input.dataPackData;
     var titleObjects = [];
-
-    if (dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c']) {
-        dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'] = this.vlocity.datapacksexpand.sortList(dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'], '%vlocity_namespace%__OmniScript__c');
+    let dataField = '%vlocity_namespace%__OmniScript__c';
+    if(!dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c']){
+        dataField = 'OmniScript__c';
+    }
+    if (dataPackData.VlocityDataPackData[dataField][0]['%vlocity_namespace%__Element__c']) {
+        dataPackData.VlocityDataPackData[dataField][0]['%vlocity_namespace%__Element__c'] = this.vlocity.datapacksexpand.sortList(dataPackData.VlocityDataPackData[dataField][0]['%vlocity_namespace%__Element__c'], '%vlocity_namespace%__OmniScript__c');
     }
     
-    var elements = dataPackData.VlocityDataPackData['%vlocity_namespace%__OmniScript__c'][0]['%vlocity_namespace%__Element__c'];
+    var elements = dataPackData.VlocityDataPackData[dataField][0]['%vlocity_namespace%__Element__c'];
```

Comment:
Can you switch it to using a variable for %vlocity_namespace%__Element__c and all the field names so that it is consistent similar to dataField where if dataField needs a namespace then they will all need a namespace, but if not then they all won't.